<launch>	
   <rosparam file="$(find cam_calib)/cfg/config.yaml" />   

    <!-- Start Rviz-->>
    <node type="rviz" name="rviz" pkg="rviz" launch-prefix="gdb -ex run --args" args="-d $(find cam_calib)/cfg/calibration.rviz"/>

    <!-- start the Realsense     -->
    <include file="$(find realsense2_camera)/launch/rs_camera.launch" >
    </include>

    <node pkg="visp_camera_calibration" name="visp_camera_calibration_calibrator" type="visp_camera_calibration_calibrator"  output="screen"/>
    
    <node pkg="cam_calib" name="cam_calib" type="cam_calib" args="camera_prefix:=/visp_camera_calibration"  output="screen">
     
    </node>

</launch>
